### PR TITLE
init: Don’t crash when a package.json is present with no name

### LINKED ISSUE #247: error:   Promise rejection: AssertionError: A name parameter is required to create a storage
Just installed polymer-cli following the instructions, then when I try it in my test/ dir, I get the following errors.

test (master)$ polymer init 
? Which starter template would you like to use? app-drawer-template
info:    Running template app-drawer-template
error:   Promise rejection: AssertionError: A name parameter is required to create a storage
error:   AssertionError: A name parameter is required to create a storage
    at new Storage (/usr/lib/node_modules/polymer-cli/node_modules/yeoman-generator/lib/util/storage.js:24:3)
    at GithubGenerator.Base._getStorage (/usr/lib/node_modules/polymer-cli/node_modules/yeoman-generator/lib/base.js:665:10)
    at GithubGenerator.Base (/usr/lib/node_modules/polymer-cli/node_modules/yeoman-generator/lib/base.js:157:22)
    at GithubGenerator (/usr/lib/node_modules/polymer-cli/lib/init/github.js:21:36)
    at Environment.instantiate (/usr/lib/node_modules/polymer-cli/node_modules/yeoman-environment/lib/environment.js:328:10)
    at Environment.create (/usr/lib/node_modules/polymer-cli/node_modules/yeoman-environment/lib/environment.js:305:15)
    at Environment.run (/usr/lib/node_modules/polymer-cli/node_modules/yeoman-environment/lib/environment.js:364:24)
    at runGenerator (/usr/lib/node_modules/polymer-cli/lib/commands/init.js:65:29)
    at /usr/lib/node_modules/polymer-cli/lib/commands/init.js:131:25

### PR DESCRIPTION
**Pull request by justinfagnani**
Fixes #247

### CODE REVIEW

File: test/init/github_test.js [hunk 1]
```diff
@@ -58,4 +59,46 @@ suite('init/github', () => {
       });
   });
 
+  test.only('works when package.json with no name is present', (done) => {
```
Comment (FredKSchott) [inline test/init/github_test.js]: `.only` snuck in

**Comment by FredKSchott** [comment]
LGTM besides that pesky `.only`

File: test/init/github_test.js (hunk 1 above)
Comment (justinfagnani) [inline test/init/github_test.js]: oops!

**Comment by justinfagnani** [comment]
Merged despite AppVeyor build, seems transient and the PR build passed.

**Pull request merged**